### PR TITLE
Enabled verifying a commit using husky in a newly created package

### LINKED ISSUE #59: Enable verifying a commit using husky in a newly created package
## Provide a description of the task

It's good to know whether a committed code passes the static analyzes. Hence, we could verify a commit using husky, similar to how we do in CKEditor 5.

### PR DESCRIPTION
**Pull request by przemyslaw-zan**
Other: Enabled verifying a commit using husky in a newly created package. Closes #59.

### CODE REVIEW

**Review by psmyrek — COMMENTED** [review]
The changes look OK... but on my Windows it does not work for unknown reasons 🤔 
After creating a new package I'm able to commit a not allowed change because `pre-commit` hook is not called at all. Calling explicitly `yarn run --silent husky-run pre-commit` works as expected.

It looks like the easy fix would be to call `npm rebuild` after all deps are installed.

It looks like this is caused by `yarn`: https://github.com/typicode/husky/issues/720#issuecomment-691532264.

**Review by psmyrek — COMMENTED** [review]
Now Husky works out of the box 👍🏻 
...but there is another issue in my env. After commiting an invalid change, ESLint is not able to load the `ckeditor5-rules` plugin, because there are 2 paths were it can be found: the `ckeditor5-rules` plugin is installed in my newly created `ckeditor5-foo` package and also it is installed in the main `ckeditor5-package-generator` package.

```bash
git commit -am 'Trying to commit an invalid change.'

husky > pre-commit (node v12.20.2)
[STARTED] Preparing...
[SUCCESS] Preparing...
[STARTED] Running tasks...
[STARTED] Running tasks for **/*.js
[STARTED] Running tasks for **/*.css
[SKIPPED] No staged files match **/*.css
[STARTED] eslint --quiet
[FAILED] eslint --quiet [FAILED]
[SUCCESS] Running tasks...
[STARTED] Applying modifications...
[SKIPPED] Skipped because of errors from tasks.
[STARTED] Reverting to original state because of errors...
[SUCCESS] Reverting to original state because of errors...
[STARTED] Cleaning up...
[SUCCESS] Cleaning up...

✖ eslint --quiet:

Oops! Something went wrong! :(

ESLint: 7.32.0

ESLint couldn't determine the plugin "ckeditor5-rules" uniquely.

- D:\Projekty\ckeditor5-package-generator\ckeditor5-foo\node_modules\eslint-plugin-ckeditor5-rules\lib\index.js (loaded in ".eslintrc.js » eslint-config-ckeditor5")
- D:\Projekty\ckeditor5-package-generator\node_modules\eslint-plugin-ckeditor5-rules\lib\index.js (loaded in "..\.eslintrc.js » eslint-config-ckeditor5")

Please remove the "plugins" setting from either config or remove either plugin installation.

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.

husky > pre-commit hook failed (add --no-verify to bypass)
```

I don't know if this is only the problem with my env, or is it a common problem? 🤔

**Comment by pomek** [comment]
@psmyrek, did you try to reproduce the problem on `#master`? Perhaps it's related to installing paths using the `file:` protocol.

**Comment by psmyrek** [comment]
On `#master`, after creating a new package and calling `./node_modules/.bin/eslint '**/.js'` from it, there is the same error.

**Comment by pomek** [comment]
On mac, everything works fine.

image

**Comment by psmyrek** [comment]
Several new findings:
- this error does not occur if I move the newly created package somewhere outside the `ckeditor5-pakcage-generator`,
- it looks like adding the `root: true` option in the `.eslintrc.js` file for a new package solves this issue in my case.

**Comment by pomek** [comment]
I remember we had the same problem with CI, and we moved the package outside the project. I couldn't reproduce the problem as I create the new package outside the project (`/Users/pomek/Projects/tmp/ckeditor5-foo`).

https://github.com/ckeditor/ckeditor5-package-generator/blob/a9e0d8fd6320d24235408255b9d523338b3faa86/scripts/ci/travis-check.js#L26

Adding `root: true` in the `.eslintrc.js` file makes sense, so let's add it.

File: packages/ckeditor5-package-generator/lib/index.js [hunk 1]
```diff
@@ -263,6 +270,40 @@ function installPackages( directoryPath, options ) {
...
+			stderr: 'inherit'
+		};
+
+		const spawnArguments = [ 'rebuild', 'husky' ];
+
+		if ( options.verbose ) {
+			spawnOptions.stdio = 'inherit';
+		}
+
+		// 'rebuild' was added to yarn in version 2, but we use yarn 1, thus only npm can be used
```
Comment (pomek) [inline packages/ckeditor5-package-generator/lib/index.js:294]:
```suggestion
		// 'rebuild' was added to yarn in version 2, but we use yarn 1, thus only npm can be used.
```

**Review by pomek — APPROVED** [review]
LGTM.

**Review by psmyrek — APPROVED** [review]
LGTM 🎉

**Pull request merged**